lib/models/rulesStore.js

```diff
@@ -97,7 +97,7 @@ function search(rule, callback) {
 
 function findAll(service, subservice, callback) {
     var criterion = {};
-
+    
```

Comment:
Remove this extra whitespace, pls.

Reply:
Fixed in 210813ba445911619199c59d729c8b724f5aef5a